dockstore-cli-integration-testing/src/test/java/io/dockstore/client/cli/GitHubAppToolIT.java

```diff
@@ -70,7 +76,12 @@ public void resetDBBetweenTests() throws Exception {
         systemOutRule.clear();
 
         // the following change the DB state
-        workflowApi.handleGitHubRelease("refs/heads/main", INSTALLATION_ID, WORKFLOW_REPO, USER_2_USERNAME);
+
+        PushPayload pushPayload = new PushPayload().ref("refs/heads/main");
```

Comment:
The only "real" change now with the bom update split out 
(and certain tests skipped)